packages/core/src/App/Containers/AccountSwitcherWallet/account-switcher-wallet.tsx

```diff
@@ -53,15 +53,19 @@ export const AccountSwitcherWallet = observer(({ is_visible, toggle }: TAccountS
             <ThemedScrollbars height={450}>
                 <AccountSwitcherWalletList wallets={dtrade_account_wallets} closeAccountsDialog={closeAccountsDialog} />
             </ThemedScrollbars>
-            <div className='account-switcher-wallet__looking-for-cfds'>
+            <div
+                className='account-switcher-wallet__looking-for-cfds'
+                onClick={handleTradersHubRedirect}
+                role='button'
+                onKeyDown={handleTradersHubRedirect}
```

Comment:
onkeydown is the same as onclick i think you can just remove it. should be fine

Reply:
Updated

Reply:
This returns the sonarcloud error 'Visible, non-interactive elements with click handlers must have at least one keyboard listener'. Reverting back to previous implementation